packages/ethereum/ethereum-light-client/src/consensus_state.rs

```diff
@@ -13,6 +13,8 @@ use crate::{error::EthereumIBCError, header::ActiveSyncCommittee, verify::BlsVer
 pub struct ConsensusState {
     /// The slot number of the finalized header
     pub slot: u64,
+    /// The execution block number
```

Comment:
Can you add a note that this is unused in the light client fully and purely exists for relayer convenience. This does feel like bad design to me. We should probably avoid coupling our relayer to the data from consensus state.

Our eventual goal is so that the relayer doesn't even deserialize the consensus state (this is how tendermint clients work). So I think we probably shouldn't do this and expect people to deserialize this.

Reply:
I do agree that we would want that, but right now we do have to do this. 
I'll keep it here for now, but I agree that the goal should be to make this unnecessary and remove it in the future 💪